hyperspy/drawing/mpl_he.py

```diff
@@ -192,6 +192,7 @@ def plot(self, **kwargs):
                 self.navigator_plot.events.closed.connect(
                     self.pointer.disconnect, [])
         self.plot_signal(**kwargs)
+        self.arrange_windows()
```

Comment:
This should be called only when there is a `navigator_plot` figure:
```suggestion
        if self.navigator_plot is not None:
            self.arrange_windows()
```